client/me/purchases/paths.js

```diff
@@ -42,15 +42,6 @@ export function confirmCancelDomain( siteName, purchaseId ) {
 	return managePurchase( siteName, purchaseId ) + '/confirm-cancel-domain';
 }
 
-export function cancelPrivacyProtection( siteName, purchaseId ) {
-	if ( process.env.NODE_ENV !== 'production' ) {
-		if ( 'undefined' === typeof siteName || 'undefined' === typeof purchaseId ) {
-			throw new Error( 'siteName and purchaseId must be provided' );
-		}
-	}
-	return managePurchase( siteName, purchaseId ) + '/cancel-privacy-protection';
```

Comment:
Looks like this route is still referenced in tests?

https://github.com/Automattic/wp-calypso/blob/18371236b33b83ce341632a9e7776690c6338296/client/lib/analytics/test/utils.js#L43
https://github.com/Automattic/wp-calypso/blob/18371236b33b83ce341632a9e7776690c6338296/client/lib/route/test/index.js#L208
https://github.com/Automattic/wp-calypso/blob/18371236b33b83ce341632a9e7776690c6338296/client/lib/route/test/index.js#L212

Reply:
Interesting, as those tests don't fail, but will remove in subsequent PRs if that's ok.